hushline/model.py

```diff
@@ -297,3 +307,106 @@ def __init__(self) -> None:
 
     def __repr__(self) -> str:
         return f"<InviteCode {self.code}>"
+
+
+# Paid tiers
+class Tier(Model):
+    __tablename__ = "tiers"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    name: Mapped[str] = mapped_column(db.String(255), unique=True)
+    monthly_amount: Mapped[int] = mapped_column(db.Integer)  # in cents USD
+    stripe_product_id = mapped_column(db.String(255), unique=True)
+    stripe_price_id = mapped_column(db.String(255), unique=True)
```

Comment:
Needs type annotations
```suggestion
    stripe_product_id: Mapped[Optional[str]] = mapped_column(db.String(255), unique=True)
    stripe_price_id: Mapped[Optional[str]] = mapped_column(db.String(255), unique=True)
```
Based on the usage and alembic migration it looks like these are optional.